ansible/roles/cockpit_plugins_cluster_setup/tasks/build_plugins.yml

```diff
@@ -0,0 +1,12 @@
+# Copyright (C) 2024, SFL (https://savoirfairelinux.com)
```

Comment:
Split the build part from the deployment part.
I preferred to use a script for the cockpit generation rather of using Ansible